h5grove/encoders.py

```diff
@@ -124,6 +120,11 @@ def encode(content: Any, encoding: Optional[str] = "json") -> Response:
             },
         )
 
+    if not is_numeric_data(content_array):
```

Comment:
Is it too permissive to allow any kind of numpy array and not just the one np.void ones? E.g. what happens with compound types?

Reply:
There doesn't seem to be any issue in the sense that the request succeeds. The front-end won't know what to do with the binary, but that's not really the back-end's fault, is it?